src/include/user/detect_test.h

```diff
@@ -5,10 +5,8 @@
  * Author: Slawomir Blauciak <slawomir.blauciak@linux.intel.com>
  */
 
-#ifndef __USER_DETECT_TEST_H__
-#define __USER_DETECT_TEST_H__
-
-#include <stdint.h>
+#ifndef __INCLUDE_UAPI_USER_DETECT_TEST_H__
```

Comment:
I think the rebase has overwritten changes from master.

Reply:
Yes, corrected.